src/js/gi-bill-comparison-tool/components/Dropdown.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+class Dropdown extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: props.defaultOption
+    };
+  }
+
+  render() {
+    if (!this.props.visible) {
+      return null;
+    }
+    return (
+      <div id={this.props.identifier + '-form'} className={this.props.className}>
+        {this.renderLabel()}
+        <select id={this.props.identifier}
+            name={this.props.identifier.replace(/\-/g, '_')}
+            alt={this.props.alt}
+            onChange={(e) => { this.props.onChange(e) && this.setState({ value: event.target.value }); }}
```

Comment:
We've been writing our form input components as [controlled components](https://facebook.github.io/react/docs/forms.html#controlled-components), where we pass the value down from a parent component (and changes back up with `onChange` hooks like you have here). You can check out how `ErrorableSelect` works to see it. It'll make it easier to migrate to our common components later if you use the same style here (or give `ErrorableSelect` a shot now).